tutorials/dapt-curation/code/requirements.txt

```diff
@@ -1,6 +1,7 @@
-arxiv
+arxiv==2.1.0
```

Comment:
Why did you add the version pin here?

Reply:
This was the error I got previously which was resolved by the above fix
```
  File "/home/ldu/Projects/supervised_fine_tuning/code/docbuilder.py", line 420, in download
    search_result = arxiv.Client().results(arxiv.Search(id_list=[article_id]))
AttributeError: 'Client' object has no attribute 'results'
```